web/src/lib/components/album-page/asset-selection.svelte

```diff
@@ -44,12 +44,15 @@
 		if (uploadAssets.length == uploadAssetsCount) {
 			// Filter assets that are already in the album
 			const assetsToAdd = uploadAssets.filter(
-				(asset) => !assetsInAlbum.some((a) => a.id === asset)
+				(asset) => !!asset && !assetsInAlbum.some((a) => a.id === asset)
```

Comment:
Why do we have this change?

Reply:
if upload progress fail, null asset id will be return in this case, so we need filter null asset from list. If not return null, we can add the rest to album because of `if (uploadAssets.length == uploadAssetsCount)` line. I don't wanna change `if (uploadAssets.length == uploadAssetsCount)` so I decide to filter null.